app/components/modal/show-request/template.hbs

```diff
@@ -0,0 +1,45 @@
+<Modal-dialog @translucentOverlay={{true}}>
```

Comment:
It's working but more consistent invoking component name should be `<ModalDialog />` without dash